fedoracommunity/widgets/package/templates/bugs_table_widget.mak

```diff
@@ -23,6 +39,11 @@
                     <option value="${rel['value']}">${rel['label']}</option>
                 % endfor
             </select>
+
+            <span class="pull-right">
+                <label for="version">Search:</label>
+                <input id="bug_search" type="text" name="search" placeholder="Bugzilla..."/>
```

Comment:
The placeholder seems odd to me, I would expect more something like: ``Search bugzilla...`` or so